### DISCUSSION TITLE
02/017-%E5%B9%95%E9%97%B4-1-%E6%97%A0%E9%97%B4%E8%BF%B7%E6%A2%A6.html

### DISCUSSION THREAD

**Discussion by giscus** [liuli-moe/to-the-stars#20] [General]
books/02/017-%E5%B9%95%E9%97%B4-1-%E6%97%A0%E9%97%B4%E8%BF%B7%E6%A2%A6
在经历了几个世纪的动荡之后，一个乌托邦式的 AI— 人类政府治理着地球，预示着后稀缺社会的来临和太空殖民的新时代。一次意外的接触却让科技更先进的敌对外星种族打破了和平，这迫使魔法少女们走出幕后，拯救人类文明。在这一切之中，志筑良子，一个普通的女孩，仰望着星空，好奇着她在宇宙中的归所。
https://tts.liuli.moe/02/017-%E5%B9%95%E9%97%B4-1-%E6%97%A0%E9%97%B4%E8%BF%B7%E6%A2%A6.html

**Comment by ArgusK17** [comment]
“一个新世界。一个更加美好的世界。那才是那些年前她们所谈论的理想。”
TTS真的让我看到了许多在现代科幻小说，甚至在这个世代的幻想文学中被丢掉的东西：对于未来的强大信心。这种上个世纪黄金年代的积极精神，因为我们社会的停滞和纷争，已经逐渐从文学作品中消失了。人们更多地去逃避现实，去读游戏和异世界小说，而不是想办法（哪怕是在幻想中）让其变得更好。

这是我当时追到最新章节时回头在幕间I写下的评论，现在看来仍旧是这样，甚至愈演愈烈。
